drivers/ufs/core/ufshcd.c

```diff
@@ -8184,7 +8184,10 @@ static void ufshcd_update_rtc(struct ufs_hba *hba)
 	 */
 	val = ts64.tv_sec - hba->dev_info.rtc_time_baseline;
 
-	ufshcd_rpm_get_sync(hba);
+	/* Skip update RTC if RPM state is not RPM_ACTIVE */
```

Comment:
Right, I suggest making it in another commit ,not all in a commit: https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v6.6.88&id=a4921b76bc9421d3838e167f6a17ea3112d8fe62